site/app/templates/admin/admin_gradeable/AdminGradeableDates.twig

```diff
@@ -1,3 +1,6 @@
+<link rel="stylesheet" href="css/flatpickr.min.css">
+<script type="text/javascript" language="javascript" src="js/flatpickr.js"></script>
```

Comment:
Instead of using these `script` tags, we use a different method for adding CSS and JS scripts to the web page (See `addInternalJS` and `addInternalCSS` and their uses in `AdminGradeableController.editPage` and `newPage` as a guide)